cmd/govim/testdata/logmessage.footer

```diff
@@ -0,0 +1,2 @@
+
+errlogmatch -start -count=0 'LogMessage callback: &protocol\.LogMessageParams\{Type:(1|2), Message:".*'
```

Comment:
Nits - Now that it's a template we could add a comment about what 1 & 2 represents so that a reader understands directly the purpose.

Reply:
Good point. Will fix that.